test/load/cfg.yml

```diff
@@ -7,6 +7,7 @@ global:
 jobs:
   - name: keptn-metrics
     jobIterations: 1
+    jobPause: 180s
```

Comment:
this blocks the test for 3min allowing Prometheus to scrape metrics for this interval